packages/utils/src/react.js

```diff
@@ -6,8 +6,8 @@ export { default as mergeRefs } from 'react-merge-refs';
 export { default as hoistNonReactStatics } from 'hoist-non-react-statics';
 
 /**
- * @param {Parameters<React.ReactChildren['toArray']>} children
- * @return {ReturnType<React.ReactChildren['toArray']>}
+ * @param {import('react').ReactNode} children
+ * @return {import('react').ReactNodeArray}
```

Comment:
The original types for this were very lazy, these new types actually correspond to how we use this function in the codebase (always taking in `children` and always casting it to an array)

Reply:
> The original types for this were very lazy
lolol. Thank you for the updates ❤️